src/App.tsx

```diff
@@ -10,31 +11,28 @@ export const tabs = [
 ];
 
 export const App: React.FC = () => {
+  const [selectedTabId, setSelectedTabId] = useState('tab-1');
+
+  const onTabSelected = (id: string) => {
+    setSelectedTabId(id);
+  };
+
   return (
     <div className="section">
       <h1 className="title">
-        Selected tab is Tab 1
+        {'Selected tab is '}
+        {tabs.find(({ id }) => id === selectedTabId)?.title}
```

Comment:
Move it to variable and use while render
```suggestion
        {`Selected tab is ${activeTab}`}
```